<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>80</y>
     <width>1081</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-6</x>
     <y>0</y>
     <width>1081</width>
     <height>71</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="simpleAttr">
    <attribute name="title">
     <string>Simple attributes</string>
    </attribute>
    <widget class="QComboBox" name="selectEV">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>5</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
	color:#000;
	border-style: 2px solid rgb(200, 200, 200);
    border-radius: 7px;
	background-color: #d0d0d0;
	padding-left: 13px;
	padding-right: 10px;
}
QComboBox::drop-down 
{
    border: 6px; /* This seems to replace the whole arrow of the combo box */
}
QComboBox::down-arrow {
   image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
    width: 9px;
    height: 9px;
	padding-right: 10px;
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>13</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Edge/Vertex:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selectStyle">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>5</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
	color: #000;
	border-style: 2px solid rgb(200, 200, 200);
    border-radius: 7px;
	background-color: #d0d0d0;
	padding-left: 13px;
	padding-right: 10px;
}
QComboBox::drop-down 
{
    border: 0px; /* This seems to replace the whole arrow of the combo box */
}
QComboBox::down-arrow {
   image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
    width: 9px;
    height: 9px;
	padding-right: 10px;
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>594</x>
       <y>13</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Style:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>13</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Attributes:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selectAttr">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>5</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
	color:#000;
	border-style: 2px solid rgb(200, 200, 200);
    border-radius: 7px;
	background-color: #d0d0d0;
	padding-left: 13px;
	padding-right: 10px;
}
QComboBox::drop-down 
{
    border: 6px; /* This seems to replace the whole arrow of the combo box */
}
QComboBox::down-arrow {
   image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
    width: 9px;
    height: 9px;
	padding-right: 10px;
}
</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="computedAttr">
    <attribute name="title">
     <string>Computed attributes</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>13</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Weight:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="centralityAttr">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>6</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
	color:#000;
	border-style: 2px solid rgb(200, 200, 200);
    border-radius: 7px;
	background-color: #d0d0d0;
	padding-left: 13px;
	padding-right: 10px;
}
QComboBox::drop-down 
{
    border: 6px; /* This seems to replace the whole arrow of the combo box */
}
QComboBox::down-arrow {
   image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
    width: 9px;
    height: 9px;
	padding-right: 10px;
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>13</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Attribute:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="centralityWeight">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>6</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
	color:#000;
	border-style: 2px solid rgb(200, 200, 200);
    border-radius: 7px;
	background-color: #d0d0d0;
	padding-left: 13px;
	padding-right: 10px;
}
QComboBox::drop-down 
{
    border: 6px; /* This seems to replace the whole arrow of the combo box */
}
QComboBox::down-arrow {
   image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
    width: 9px;
    height: 9px;
	padding-right: 10px;
}
</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>809</x>
     <y>70</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Coordinates</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
